classes/Combination.php

```diff
@@ -114,6 +114,31 @@ class CombinationCore extends ObjectModel
         ],
     ];
 
+    /**
+     * @param null $id
+     * @param null $id_lang
+     * @param null $id_shop
+     */
+    public function __construct($id = null, $id_lang = null, $id_shop = null)
+    {
+        parent::__construct($id, $id_lang, $id_shop);
+        $this->loadStockData();
+    }
+
+    /**
+     * Fill the variables used for stock management.
+     */
+    public function loadStockData()
+    {
+        if (false === Validate::isLoadedObject($this)) {
+            return;
+        }
+
+        // Default product quantity is available quantity to sell in current shop
+        $this->quantity = StockAvailable::getQuantityAvailableByProduct($this->id_product, $this->id);
+        $this->location = StockAvailable::getLocation($this->id);
```

Comment:
```suggestion
        $this->location = StockAvailable::getLocation($this->id_product, $this->id);
```
woups just noticed this one 😅